### ISSUE TITLE
Config errors cause an exception to be thrown

### ISSUE THREAD

**Issue by yamikuronue** [SockDrawer/SockBot#161]
It would be politer to pass an error to the callback

**Comment by AccaliaDeElementia** [comment]
should be resolved by d059e54

Please confirm that this is the case now.

**Comment by yamikuronue** [comment]
```

Welcome to SockMon! You are currently configured to use the default config. If you want to get started, type 'start'. To change the config file, try 'config'. To exit, type 'exit'.
start
[11:49:25] Starting SockBot 0.0.1 undefined
Server started! You can access it at localhost:8000
[11:49:26] Login Failed: Unknown Login Failure
ERROR starting bot: Unknown Login Failure

```

I'd say that's a success :D
